pkgs/development/python-modules/dask-awkward/default.nix

```diff
@@ -27,7 +27,7 @@
 
 buildPythonPackage rec {
   pname = "dask-awkward";
-  version = "2024.9.0";
+  version = "2024.11.0";
```

Comment:
Isn't it odd how the hash didn't change? The upstream diff is not empty.
Also in https://github.com/NixOS/nixpkgs/pull/359521/files

Reply:
Indeed, this is weird. I have double-checked and the hash has indeed stayed the same.

Reply:
Triple checked:
```
❯ nix-prefetch-github dask-contrib dask-awkward --rev 2024.11.0
{
    "owner": "dask-contrib",
    "repo": "dask-awkward",
    "rev": "f5b370224646d6fee14826475f052888c9bb142d",
    "hash": "sha256-4CwixPj0bJHVjnwZ7fPkRdiDHs8/IzvNlwSPynXvcAo="
}
```

Reply:
> The upstream diff is not empty.

It is empty: https://github.com/dask-contrib/dask-awkward/compare/2024.9.0..2024.11.0 (two dots not three).

Reply:
Yeah, looks like they rebased some commits from 2024.9.0, so they show up as "committed yesterday". I propose we skip the update and stay away from the new tag.

Reply:
Ok good for me. Closing then.